src/number_theory/number_field.lean

```diff
@@ -4,23 +4,27 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Ashvni Narayanan, Anne Baanen
 -/
 
-import algebra.field.basic
-import data.rat.basic
-import ring_theory.algebraic
 import ring_theory.dedekind_domain.integral_closure
-import ring_theory.integral_closure
-import ring_theory.polynomial.rational_root
+import algebra.char_p.algebra
+import analysis.complex.polynomial
```

Comment:
It's maybe a good idea to create a new file with relations of a number field with `ℂ`. The reason is that (I think) `analysis.complex.polynomial` imports quite a lot of analytic stuff, making this file not "algebraic" anymore.